packages/transport/package.json

```diff
@@ -67,8 +67,6 @@
         "@trezor/protocol": "workspace:*",
         "@trezor/utils": "workspace:*",
         "cross-fetch": "^4.0.0",
-        "long": "^4.0.0",
-        "protobufjs": "7.4.0",
```

Comment:
Seems that protobufjs shouldn't have gone away.

Reply:
hmm yes, there is still one more runtime usecase in packages/transport. maybe in the future it should be factored out